addons/stock/views/res_config_settings_views.xml

```diff
@@ -135,6 +135,16 @@
                                         <div class="text-muted">Use GS1 nomenclature datamatrix whenever barcodes are printed for lots and serial numbers.</div>
                                     </div>
                                 </div>
+                                <div class="row mt-2" invisible="not group_stock_production_lot or not group_stock_tracking_lot">
+                                    <span class="o_form_label">Scannable package contents</span>
+                                    <div class="col-12">
+                                        <div class="text-muted">In package content, generate a QR code containing all serials contained</div>
```

Comment:
```suggestion
                                        <div class="text-muted">Generate a QR code containing all serials contained in package contents</div>
```

Also sorry I forgot to respond to your other comment where you suggested:

`If defined, a QR code containing all serial numbers contained in package will be generated, using the defined character(s) to separate each numbers`

I think adding the `If a separator is defined,` to the beginning of the sentence is a good idea.